src/components/Overview/Overview.react.js

```diff
@@ -216,18 +246,20 @@ SUSI is intelligent to identify and answer your question in your language.
             <img src={susiSkill} alt='Skills' className='skillWiki'/>
           </div>
         </div>
+        </ScrollAnimation>
+
+
+        <ScrollAnimation animateIn="fadeIn" offset="300" duration="3">
         <div className="section_copy safe_and_secure">
           <div className="conversation__description">
 
             <div className="description__heading">Safe and secure.</div>
             <p className="description__text"><b>SUSI.AI</b> is <b>
               <Link style={{ textDecoration: 'none', color: '#000' }}
-                target="_blank" to="https://github.com/fossasia?utf8=%E2%9C%93&q=susi">Open Source</Link></b>. The code is
-                    always available for security reviews and can be improved by
-                    anyone with the knowledge and understanding online.</p>
+                target="_blank" to="https://github.com/fossasia?utf8=%E2%9C%93&q=susi">Open Source</Link></b>. The code is always available for security reviews and can be improved by anyone with the knowledge and understanding online.</p>
                     <div className="opensource-logos">
                         <span className="opensource">
-                          <Link to="https://opensource.org/" target="_blank">
+                          <Link to="https://opensource.org/" target="_blank">""
```

Comment:
Same here.

Reply:
Extremely sorry for the inconvenience. Fixing it right away.